data/oecd/oecd_tariffs.ipynb

```diff
@@ -0,0 +1,231 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "784fc8c9-fe40-4865-bece-0fdee54653d7",
+   "metadata": {},
+   "source": [
+    "# OECD renewable energy feed-in tariffs\n",
+    "\n",
+    "The Organization for Economic Co-operation and Development ([OECD](https://www.oecd.org/)) publishes data about the [renewable energy feed-in tariffs](https://stats.oecd.org/Index.aspx?DataSetCode=RE_FIT) (FITs): \n",
+    "\n",
+    "> Feed-in tariffs (FITs) are prevalent support policies for scaling up renewable electricity capacity. They are market-based economic instruments, which typically offer long-term contracts that guarantee a price to be paid to a producer of a pre-determined source of electricity per kWh fed into the electricity grid.\n",
+    ">\n",
+    "> This dataset provides FITs values derived in a manner that is comparable across countries, years and renewable energy sub-sectors. The data include country-level values on the tariff (in USD/kWh), and length of the awarded power-purchasing agreement. The dataset covers seven renewable electricity sub-sectors: wind, solar photovoltaic (concentrated solar power is excluded), geothermal, small hydro, geothermal, marine, biomass and waste.\n",
+    ">\n",
+    "> This dataset is fully maintained by the OECD secretariat, drawing on government sources (e.g. websites, official documents) and information from research institutes. In addition, the data are cross-checked against other renewable energy policy databases (e.g. REN21, IEA/IRENA, [OECD PINE database](http://www.oecd.org/environment/indicators-modelling-outlooks/policy-instrument-database/)).\n",
+    ">\n",
+    "> For further details on the methodology and examples of applications of this dataset, please consult [the documentation](https://stats.oecd.org/wbos/fileview2.aspx?IDFile=7e7f7564-1046-4932-bfad-d24f2a679f15).\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ea7ce933-c4e1-4d8d-afc6-f8c51a3f4619",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import pandas as pd\n",
+    "import datetime as dt\n",
+    "import pandas_datareader.data as web"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "69e6665e-e692-4527-9464-88890d26b216",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start_time = dt.datetime(2000, 1, 1)\n",
+    "end_time = dt.datetime(2022, 10, 1)\n",
+    "df = web.DataReader('RE_FIT', 'oecd', start_time, end_time)\n",
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7a21cf51-e606-43ec-8fc6-50693b080305",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Move Country and Variable to rows\n",
+    "df = df.stack(level=[0, 1]).reset_index()\n",
```

Comment:
The loaded DataFrame has multi index columns: Country, Variable and Source.  I moved `Country` to rows here, and remove `Variable` below to keep only tariff data in a simple table.